submit/sub.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+"""
+Submit job to CPG
+Suggested and programmed by zhili<zhilizheng@outlook.com>
+Tested and run by Zhenqiao
+License: MIT
+Please read the document from analysis-runner
+https://anaconda.org/cpg/analysis-runner
+"""
+
+from cpg_utils.hail_batch import output_path
+from cpg_utils.hail_batch import remote_tmpdir
+
+from analysis_runner.git import (
+  prepare_git_job,
+  get_repo_name_from_current_directory,
+  get_git_commit_ref_of_current_repository
+)
+
+import hailtop.batch as hb
+
+import click
+import os
+
+BILLING_PROJECT = os.getenv('HAIL_BILLING_PROJECT')
+assert BILLING_PROJECT
+
+def makeBatch():
+    """
+    make the batch backend
+    """
+    # Initializing Batch
+    backend = hb.ServiceBackend(
+        billing_project=BILLING_PROJECT, remote_tmpdir=remote_tmpdir()
+    )
+    return hb.Batch(backend=backend, default_image=os.getenv('DRIVER_IMAGE'))
+
+
+@click.command()
```

Comment:
This is effectively what the analysis-runner does for you (except saving the log to gcs). Do you have a specific use-case for this script that isn't addressed by the analysis-runner?

Reply:
It is written by a friend to help me submitting jobs in a manner similar as submitting jobs on institutional clusters. It will be great if I can keep using these tools for some time, until I am more familiar with Hail batch &/ dataproc :)

Reply:
Note that `bcftools` can read from GCS directly, so you don't need the mount option here. As @illusional is pointing out, at this point you really don't get any additional functionality through this compared to the vanilla analysis-runner -- it just adds another level of indirection and code to maintain.